HerPublicWebsite.BusinessLogic/ExternalServices/EpbEpc/EpbEpcApi.cs

```diff
@@ -47,6 +52,11 @@ public async Task<EpcDetails> EpcFromUprnAsync(string uprn)
                 var response = await HttpRequestHelper.SendGetRequestAsync<EpbEpcDto>(parameters);
                 return response.Data.Assessment.Parse();
             }
+            catch (ApiException e) when (e.StatusCode == HttpStatusCode.NotFound)
+            {
+                logger.LogInformation("EPB EPC request could not find any results for UPRN {}", uprn);
```

Comment:
I was debating between Debug and Info here... I don't know if we've got any particular guidance to follow on that.

Reply:
Info seems fine